src/components/input-field/index.md.njk

```diff
@@ -1,6 +1,6 @@
 ---
-title: Text input
-description: Help users enter information with the text input component
+title: Input field
+description: Help users enter information with the input field component
 section: Components
 aliases: text box, text field, input field, text entry box
```

Comment:
could we change the 'input field' alias to 'text input' now?

Reply:
Good catch. Do you think we should add 'number input' as an alias now too? 🤔

Reply:
Maybe, I'll leave that for you to decide 🤔

Reply:
I've gone with adding it. This will go through a content review too so we'll look at it there.